cmd/minibroker/main.go

```diff
@@ -79,6 +82,18 @@ func main() {
 	})
 	defer klog.Flush()
 
+	if options.ClusterDomain == "" {
+		resolvConf, err := os.Open("/etc/resolv.conf")
+		if err != nil {
+			klog.Fatalln(err)
+		}
```

Comment:
We should do a `defer resolvConfg.Close()` here instead (so that we'll do it correctly in the `klog.Fatalln()` case).

Yes, I'm aware that the file handle will be closed on process exit anyway.

Reply:
Exactly, we don't need a defer for closing.

Reply:
But we do if we ever end up modifying this function later. If the language has a good construct for this, we should use it. (Otherwise it'd be `try`/`finally` or similar).

Reply:
Fair enough...

Reply:
Fixed in b82d19f.

Reply:
Let's be clear, the `.Close()` that matter is not the `defer` one because the `defer` only gets called on Minibroker termination.